base/array.jl

```diff
@@ -982,6 +991,10 @@ julia> prepend!([3],[1,2])
  2
  3
 ```
+
+Use [`unshift!`](@ref) to add individual items to `collection` which are not already
+themselves in another collection. The result is of the preceding example is equivalent to
+`unshift!([1, 2, 3], 4, 5, 6)`.
```

Comment:
The previous example (`prepend!([3],[1,2])` I believe) actually has a different result.  And there is a duplicate "is" in the above sentence.